### PR TITLE
Create new topic 'neo'

### PR DESCRIPTION
**Pull request by PeterLinX**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:
NEO is a one of the most popular blockchain in the world with very mature developer community. You may find more informations on coin market [here](https://coinmarketcap.com/). We found less than 100 repositories under this topic, however, there should be over 300 repositories out there. We think it's time to create this topic.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/neo/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+created_by: Hongfei Da, Erik Zhang
+display_name: NEO
+logo: neo.png
+released: July 15, 2016
+short_description: A distributed network for the Smart Economy.
+topic: neo
+related: blockchain, cryptocurrency
+url: https://www.neo.org/
+github_url: https://github.com/neo-project
+documentation_url: http://docs.neo.org/
```
Comment (GloverDonovan) [inline topics/neo/index.md]: This is not a valid field. See: https://github.com/github/explore/blob/master/docs/API.md

Once you remove this line (assuming no other errors), the Travis CI build should pass.

File: topics/neo/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+created_by: Hongfei Da, Erik Zhang
+display_name: NEO
+logo: neo.png
+released: July 15, 2016
+short_description: A distributed network for the Smart Economy.
+topic: neo
+related: blockchain, cryptocurrency
+url: https://www.neo.org/
+github_url: https://github.com/neo-project
+wikipedia_url: http://docs.neo.org/
```
Comment (GloverDonovan) [inline topics/neo/index.md]: Change this URL to https://en.wikipedia.org/wiki/NEO_(cryptocurrency) ?

Comment (PeterLinX) [reply topics/neo/index.md]: sure, no problem. I have updated logo and wiki url.

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @PeterLinX!

**Pull request merged**